src/baseCanvas/Minimap.tsx

```diff
@@ -4,6 +4,8 @@ import { PositionAndSize } from "@/baseCanvas/Canvas";
 import { Props as BaseProps, BaseCanvas } from "./Canvas";
 
 export interface Props extends BaseProps {
+  imageWidth: number;
```

Comment:
Do we ever use one of these without the other? Should they not just be a single prop holding both numbers?

Reply:
turns out imageWidth and imageHeight weren't needed. thanks!